string type
---
string type
webrtc_ros/MessageInfo[] infos
